README.md

```diff
@@ -1375,6 +1375,15 @@ _You can enable the following settings in Xcode by running [this script](resourc
 
   </details>
 
+* <a id='no_direct_standard_out'></a>(<a href='#no_direct_standard_out'>link</a>) **Prefer dedicated logging systems like [`os_log`](https://developer.apple.com/documentation/os/logging) or [`swift-log`](https://github.com/apple/swift-log) over writing directly to standard out using `print(…)`, `logDebug(…)`, or `dump(…)`.**
```

Comment:
This part of the `swift-log` README is interesting.
<img width="1133" alt="Screen Shot 2021-04-22 at 6 03 27 PM" src="https://user-images.githubusercontent.com/1791049/115803136-1f914200-a395-11eb-83bd-b911b74c75b3.png">